src/pages/_app.tsx

```diff
@@ -20,6 +20,7 @@ import theme from '../theme';
 const { baseUrl, isProduction, recaptchaKey } = config;
 
 Router.events.on('routeChangeComplete', url => pageview(url));
+Router.events.on('hashChangeComplete', url => pageview(url));
```

Comment:
Are this event listeners required for something? With this new logic, i don't think we need, and we should avoid to add generic event listeners, as much as we can.